src/nextgis_connect/detached_editing/detached_layer.py

```diff
@@ -51,13 +51,15 @@ class DetachedLayer(QObject):
     __updated_geometries: Dict[QgsFeatureId, str]
     __deleted_features: Dict[QgsFeatureId, QgsFeature]
 
+    __is_layer_changed: bool = False
```

Comment:
Please move this initialization in the constructor, otherwise the value will be assigned not to the object, but to the class